examples/widgets/textbox.py

```diff
@@ -32,3 +38,15 @@ def submit(text):
 text_box.on_submit(submit)
 
 plt.show()
+
+#############################################################################
+#
+# ------------
+#
+# References
+# """"""""""
+#
+# The use of the following functions, methods, classes and modules is shown
+# in this example:
+
+from matplotlib.widgets import TextBox
```

Comment:
Please add `./examples/widgets/textbox.py: E402` to `.flake8`. So that this is not flagged as a style error.

Reply:
Ah I didn't turn on linting. Fixed as you suggested.